plugins/woocommerce-blocks/packages/checkout/filter-registry/test/index.js

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import { renderHook } from '@testing-library/react-hooks';
+import { renderHook } from '@testing-library/react';
```

Comment:
I updated all the testing lib dependencies. the renderhook is now part of the core lib